packages/kotti-table/src/TableRow.js

```diff
@@ -145,20 +158,39 @@ export default {
 			$event.stopPropagation()
```

Comment:
This should be a `@select.stop`.

Reply:
but this is a js file not a view file so I have to handle events manually